opcua/client/client.py

```diff
@@ -58,7 +58,11 @@ def run(self):
             if self._dostop:
                 break
             self.logger.debug("renewing channel")
-            self.client.open_secure_channel(renew=True)
+            try:
+                self.client.open_secure_channel(renew=True)
+            except ConnectionError as e:
+                self.logger.debug('keepalive failed: {:s}'.format(str(e)))
+                break
```

Comment:
I am not really sure what it helps. What do you want? simply a cleaner output in case of crash? or do you want to detect connection error sooner?

Reply:
There is no negative impact. The only task of the KeepAlive thread is to keep the connection alive, but I was '''annoyed''' by the backtrace in the logs every once in a while.
I'll remove the ConnectionError and handle the timeout in the KeepAlive then.